docs/query-builder.txt

```diff
@@ -221,6 +221,13 @@ value greater than ``8.5`` and a ``year`` value of less than
    :start-after: begin query andWhere
    :end-before: end query andWhere
 
+.. tip::
+
+   In v5.1, you can use the arrow (``->``) or dot (``.``) notation
+   to access nested fields in a query filter. The preceding example
+   uses the dot notation to query the ``imdb.rating`` nested field, but
```

Comment:
Nit: I think "arrow notation" and "dot notation" without "the" is more common? Disregard if this is the laravel-specific terminology though
```suggestion
   In v5.1, you can use arrow (``->``) or dot (``.``) notation
   to access nested fields in a query filter. The preceding example
   uses dot notation to query the ``imdb.rating`` nested field, but
```